tests/components/textbox.test.jsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I was hoping to make these tests more comprehensive, but I had difficulty getting them to properly simulate user actions, and I was coming up against the release deadline. The behaviour that this PR fixes is perhaps best tested with Selenium.